src/koa-middleware/optionalAuthentication.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Please add a comment describing this file why do we need it and what is it about.